.github/workflows/publish.yml

```diff
@@ -182,10 +182,19 @@ jobs:
           rm ./packages-macos/tt*checksums.txt
           mv checksums.txt ./packages-macos/
 
+      - name: Set GoReleaser flags
+        id: set-goreleaser-flags
+        run: |
+          if ${{ startsWith(github.ref, 'refs/tags') }} ; then
+            echo "GORELEASER_FLAGS=--skip-validate" >> $GITHUB_OUTPUT
+          else
+            echo "GORELEASER_FLAGS=--snapshot" >> $GITHUB_OUTPUT
+          fi
+
```

Comment:
Could you please examplain the motivation? (preferably in a commit)

Reply:
Added some details to the commit message of the changes.